yt_dlp/YoutubeDL.py

```diff
@@ -25,15 +25,17 @@
 from .cache import Cache
 from .compat import urllib  # isort: split
 from .compat import compat_os_name, compat_shlex_quote
+from .compat.compat_utils import get_package_info
 from .cookies import load_cookies
+from .dependencies import available_dependencies
```

Comment:
This does not delay import the dependencies anymore. While its not a lot of overhead, it should still come with a bit no?